app/views/entities/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render 'devise/shared/navbar' %>
+<div class="group-new">
+  <%= form_with model: @entity, html: {'data-turbo' => "false"}, url: entities_path do |form| %>
+    <div class="form-new-title field">
+      <%= form.label :name %>
+      <%= form.text_field :name, required: true %>
+    </div>
```

Comment:
- [ ] Kindly add go back button in add transaction page, so that users when the user clicks on the "Back" button (<), the user navigates to the transactions page for that category